wgengine/router/router_linux.go

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
+//go:build linux
```

Comment:
this is implied by the _linux suffix on the file name, and doesn't need to be added